docs/content/config/index.md

```diff
@@ -0,0 +1,79 @@
+
+## The `.env` file
+
+The `.env` file should be places in the root of the project and contains key value pairs of environment variables, and the values you configured them with. This can for example look like this:
+
+```ini
+HD_DOMAIN="http://localhost"
+HD_MEDIA_BACKEND="filesystem"
+HD_MEDIA_BACKEND_FILESYSTEM_UPLOAD_PATH="uploads/"
+HD_DATABASE_DIALECT="sqlite"
+HD_DATABASE_STORAGE="./hedgedoc.sqlite"
+```
+
+We also provide an `.env.example` file containing a minimal configuration in the root of the project. This should help you to write your own configuration.
+
+!!! warning  
+    The minimal configuration provided in `.env.example` is exactly that: minimal. It will let you start HedgeDoc, but it is **not** meant to be used in production without prior changes.
+
+## General
+
+| environment variable     | default   | example                    | description                                                                                       |
+|--------------------------|-----------|----------------------------|---------------------------------------------------------------------------------------------------|
+| `HD_DOMAIN`              | -         | https://md.example.com     | The URL the HedgeDoc instance runs on.                                                            |
+| `PORT`                   | 3000      |                            | The port the HedgeDoc instance runs on.                                                           |
+| `HD_RENDERER_ORIGIN`     | HD_DOMAIN |                            | The URL the renderer runs on. If omitted this will be same as `HD_DOMAIN`.                        |
+| `HD_LOGLEVEL`            | warn      |                            | The loglevel that should be used. Options are `error`, `warn`, `info`, `debug` or `trace`.        |
+| `HD_FORBIDDEN_NOTE_IDS`  | -         | notAllowed, alsoNotAllowed | A list of note ids (separated by `,`), that are not allowed to be created or requested by anyone. |
+| `HD_MAX_DOCUMENT_LENGTH` | 100000    |                            | The maximum length of any one document. Changes to this will impact performance for your users.   |
+
+## Authentication
+
+**ToDo:** Add Authentication docs
+
+## Customization
+
+| environment variable  | default | example                         | description                                                                                                                                                                |
+|-----------------------|---------|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `HD_CUSTOM_NAME`      | -       | DEMO Corp                       | The text will be shown in the top right corner in the editor and on the intro page. If you also configured a custom logo this will be used as the alt text of the logo.    |
```

Comment:
```suggestion
| `HD_CUSTOM_NAME`      | -       | DEMO Corp                       | The text will be shown in the top right corner in the editor and on the intro page. If you also configure a custom logo, this will be used as the alt text of the logo.    |
```